cdc/owner/owner.go

```diff
@@ -780,6 +780,20 @@ func (o *ownerImpl) updateGCSafepoint(
 	return nil
 }
 
+// ignoreFailedFeedWhenGC checks if a failed changefeed should be ignored
+// when calculating the gc safepoint of the associated upstream.
+func (o *ownerImpl) ignoreFailedFeedWhenGC(
```

Comment:
use changefeed instead

Reply:
switch to `ignoreFailedChangeFeedWhenGC`